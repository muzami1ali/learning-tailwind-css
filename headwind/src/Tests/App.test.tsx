import App from '../Components/App';
import renderer from 'react-test-renderer';
// Snapshot test for App
test('App snapshot test', () => {
  const component = renderer.create(
      <App />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});