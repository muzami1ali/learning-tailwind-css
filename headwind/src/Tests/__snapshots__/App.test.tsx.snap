// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<div
  className="flex"
>
  <div
    className="fixed top-0 left-0 h-screen w-16 flex flex-col bg-gray-900 text-white shadow-lg"
  >
    <div
      className="sidebar-icon group"
    >
      <svg
        fill="currentColor"
        height={28}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 384 512"
        width={28}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M216 23.86c0-23.8-30.65-32.77-44.15-13.04C48 191.85 224 200 224 288c0 35.63-29.11 64.46-64.85 63.99-35.17-.45-63.15-29.77-63.15-64.94v-85.51c0-21.7-26.47-32.23-41.43-16.5C27.8 213.16 0 261.33 0 320c0 105.87 86.13 192 192 192s192-86.13 192-192c0-170.29-168-193-168-296.14z"
        />
      </svg>
      <span
        className="sidebar-tooltip group-hover:scale-100"
      >
        tooltip ðŸ’¡
      </span>
    </div>
    <div
      className="sidebar-icon group"
    >
      <svg
        fill="currentColor"
        height={32}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 16 16"
        width={32}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M8 4a.5.5 0 0 1 .5.5v3h3a.5.5 0 0 1 0 1h-3v3a.5.5 0 0 1-1 0v-3h-3a.5.5 0 0 1 0-1h3v-3A.5.5 0 0 1 8 4z"
        />
      </svg>
      <span
        className="sidebar-tooltip group-hover:scale-100"
      >
        tooltip ðŸ’¡
      </span>
    </div>
    <div
      className="sidebar-icon group"
    >
      <svg
        fill="currentColor"
        height={20}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 16 16"
        width={20}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M5.52.359A.5.5 0 0 1 6 0h4a.5.5 0 0 1 .474.658L8.694 6H12.5a.5.5 0 0 1 .395.807l-7 9a.5.5 0 0 1-.873-.454L6.823 9.5H3.5a.5.5 0 0 1-.48-.641l2.5-8.5z"
        />
      </svg>
      <span
        className="sidebar-tooltip group-hover:scale-100"
      >
        tooltip ðŸ’¡
      </span>
    </div>
    <div
      className="sidebar-icon group"
    >
      <svg
        fill="currentColor"
        height={20}
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width={20}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M451.4 369.1C468.7 356 480 335.4 480 312c0-39.8-32.2-72-72-72h-14.1c13.4-11.7 22.1-28.8 22.1-48 0-35.3-28.7-64-64-64h-5.9c3.6-10.1 5.9-20.7 5.9-32 0-53-43-96-96-96-5.2 0-10.2.7-15.1 1.5C250.3 14.6 256 30.6 256 48c0 44.2-35.8 80-80 80h-16c-35.3 0-64 28.7-64 64 0 19.2 8.7 36.3 22.1 48H104c-39.8 0-72 32.2-72 72 0 23.4 11.3 44 28.6 57.1C26.3 374.6 0 404.1 0 440c0 39.8 32.2 72 72 72h368c39.8 0 72-32.2 72-72 0-35.9-26.3-65.4-60.6-70.9zM192 256c17.7 0 32 14.3 32 32s-14.3 32-32 32-32-14.3-32-32 14.3-32 32-32zm159.5 139C341 422.9 293 448 256 448s-85-25.1-95.5-53c-2-5.3 2-11 7.8-11h175.4c5.8 0 9.8 5.7 7.8 11zM320 320c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z"
        />
      </svg>
      <span
        className="sidebar-tooltip group-hover:scale-100"
      >
        tooltip ðŸ’¡
      </span>
    </div>
  </div>
</div>
`;
